ChangeLog

```diff
@@ -16,6 +16,9 @@
  * HSICEstimatorTargetSensitivity
  * HSICUStat
  * HSICVStat
+ * IterativeExtrema
+ * IterativeMoments
+ * IterativeThresholdExceedance
```

Comment:
add a line in the "major changes" section too

Reply:
Oups : of course. Done.